docs/understanding-docker.md

```diff
@@ -17,273 +17,282 @@ weight = -90
+the foreground in interactive mode (`-i`), and to run the `/bin/bash` command.
 
     $ docker run -i -t ubuntu /bin/bash
 
-The Docker Engine client is launched using the `docker` binary with the `run` option
-running a new container. The bare minimum the Docker client needs to tell the
-Docker daemon to run the container is:
 
-* What Docker image to build the container from, for example, `ubuntu`
-* The command you want to run inside the container when it is launched,
-for example,`/bin/bash`
+When you run this command, Docker Engine does the following:
+
+1. **Pulls the `ubuntu` image:** Docker Engine checks for the presence of the
+    `ubuntu` image. If the image already exists locally, Docker Engine uses it for
+    the new container. Otherwise, then Docker Engine pulls it from
+    [Docker Hub](https://hub.docker.com).
+
+1. **Creates a new container:** Docker uses the image to create a container.
+
+1. **Allocates a filesystem and mounts a read-write _layer_:** The container is
+    created in the file system and a read-write layer is added to the image.
+
+1. **Allocates a network / bridge interface:** Creates a network interface that
+    allows the Docker container to talk to the local host.
 
-So what happens under the hood when we run this command?
+1. **Sets up an IP address:** Finds and attaches an available IP address from a
+    pool.
 
-In order, Docker Engine does the following:
+1. **Executes a process that you specify:** Executes the `/bin/bash` executable.
 
-- **Pulls the `ubuntu` image:** Docker Engine checks for the presence of the `ubuntu`
-image. If the image already exists, then Docker Engine uses it for the new container.
-If it doesn't exist locally on the host, then Docker Engine pulls it from
-[Docker Hub](https://hub.docker.com).
-- **Creates a new container:** Once Docker Engine has the image, it uses it to create a
-container.
-- **Allocates a filesystem and mounts a read-write _layer_:** The container is created in
-the file system and a read-write layer is added to the image.
-- **Allocates a network / bridge interface:** Creates a network interface that allows the
-Docker container to talk to the local host.
-- **Sets up an IP address:** Finds and attaches an available IP address from a pool.
-- **Executes a process that you specify:** Runs your application, and;
-- **Captures and provides application output:** Connects and logs standard input, outputs
-and errors for you to see how your application is running.
+1. **Captures and provides application output:** Connects and logs standard input,
+    outputs and errors for you to see how your application is running, because you
+    requested interactive mode.
 
-You now have a running container! Now you can manage your container, interact with
-your application and then, when finished, stop and remove your container.
+After the container is running, you can manage and interact with it, and eventually
+stop and remove it.
```

Comment:
:-) Docker has a characteristically frivolous, energetic, and animated
personality. I don't think we should be the ones to drain that away.

On Fri, Sep 2, 2016 at 1:27 PM, Misty Stanley-Jones <
notifications@github.com> wrote:

> In docs/understanding-docker.md
> <https://github.com/docker/docker/pull/26256#discussion_r77406764>:
>
> >
> > -You now have a running container! Now you can manage your container, interact with
> > -your application and then, when finished, stop and remove your container.
> > +After the container is running, you can manage and interact with it, and eventually
> > +stop and remove it.
>
> Done (but I really dislike exclamation points!)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/docker/pull/26256/files/d3c45a470519421f9e0bfc8496544bb8714547ba#r77406764>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALHuAz1mWml9WZ6OngYq2sZN8cCrnYrAks5qmIabgaJpZM4JzIGC>
> .
>